kitsune/search/v2/search.py

```diff
@@ -114,7 +115,13 @@ def get_highlight_fields(self):
         ]
 
     def get_filter(self):
-        filters = [DSLQ("term", _index=self.get_index())]
+        # Add default filters:
+        filters = [
+            # limit scope to the Wiki index
+            DSLQ("term", _index=self.get_index()),
+            # exclude archived articles
+            DSLQ("term", **{"is_archived.{self.locale}": False}),
```

Comment:
The `f` is missing off the start of this f-string, it should be `f"is_archived.{self.locale}"`